src/app.js

```diff
@@ -21,13 +21,9 @@ app.use(logger("dev"));
 app.use(express.static("public"));
 app.use(favicon(path.join(__dirname, "/public/favicon.ico")));
 
-app.get("/", (req, res) => {
-  res.send('<h2>Welcome!</h2><a href="#">Documentation</a>');
-});
-
```

Comment:
Oh I'm sorry! I didn't mean that you needed to remove this page! It's totally okay to break tests if you're intentionally changing the behavior. My comment was just that we'll need to update the test to reflect the new desired behavior. And I can do that if you want to add this back. Sorry for the confusion!

Reply:
No worries, I am going to add that back and change the test itself this time.

Reply:
Sounds good! 👍